app/src/main/java/io/github/droidkaigi/confsched2018/data/api/response/mapper/NoticeDataMapperExt.kt

```diff
@@ -15,7 +15,7 @@ fun List<io.github.droidkaigi.confsched2018.data.api.response.Post>.toFeeds():
                     "tutorial" -> Post.Type.Tutorial
                     "notification" -> Post.Type.Notification
                     "alert" -> Post.Type.Alert
-                    "enquete" -> Post.Type.Enquete
+                    "feedback" -> Post.Type.Feedback
```

Comment:
Sorry, we should use "enquete" for this string. 🙇 
Post.Type.Feedback is ok.But this string should be "enquete" 🙇

Reply:
OK, I'll revert 🏃